code_samples/front/shop/list_products/src/Controller/ProductListController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Controller;
+
+use Ibexa\Contracts\ProductCatalog\Values\Product\ProductQuery;
+use Ibexa\ProductCatalog\Local\Repository\ProductService;
```

Comment:
\\Ibexa\\Contracts\\ProductCatalog\\ProductServiceInterface should be used here.